content/articles/inbo_software/index.md

```diff
@@ -108,6 +114,7 @@ Miscellaneous (mixed content) | [inborutils][inborutils]
 - **R package [inlatools][inlatools]**: provides a set of functions which can be useful to diagnose [INLA models](http://www.r-inla.org/): calculating Pearson residuals, simulation based checks for over- or underdispersion, simulation based checks for the distribution, visualising the effect of the variance or precision on random effects (random intercept, first order random walk, second order random walk). The functions can be useful to choose sensible priors and diagnose the fitted model.
 - **R package [multimput][multimput]**: an R package that assists with analysing datasets with missing values using multiple imputation.
 - **Python package [niche_vlaanderen][niche_vlaanderen]**: Python package to run the NICHE Vlaanderen model. Based on calculated abiotic properties of the location, NICHE Vlaanderen determines whether certain vegetation types can develop. An additional flooding module allows the user to test whether the predicted vegetations are compatible with a particular flooding regime. The package is a redevelopment of an existing ArcGIS plugin in Python, without external non-open source dependencies.
+- **R package [qgisprocess][qgisprocess]**: R interface to the geoprocessing algorithms of [QGIS](https://qgis.org/en/site/) and other providers that can interface with QGIS (e.g. [GDAL](https://gdal.org/index.html), [grass](https://grass.osgeo.org/), [SAGA](https://saga-gis.sourceforge.io/en/index.html) and [whitebox tools](https://www.whiteboxgeo.com/geospatial-software/)).
```

Comment:
Thanks.

```suggestion
- **R package [qgisprocess][qgisprocess]**: R interface to the geoprocessing algorithms of [QGIS](https://qgis.org/en/site/) and other providers that can interface with QGIS (e.g. [GDAL](https://gdal.org/index.html), [GRASS GIS](https://grass.osgeo.org/), [SAGA](https://saga-gis.sourceforge.io/en/index.html) and [WhiteboxTools](https://www.whiteboxgeo.com/geospatial-software/)).
```

The match with 'model fitting' seems less ideal. Maybe expand the title 'make graphs' to 'data processing & visualization' and move it there? Or put it together with {inbospatial} below some geospatial title (although such title may fit harder with the steps envisaged here).

Reply:
'General', 'miscellaneous', ... are also options of course.